src/views/partials/cancelSubscriptionSurvey.js

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getMessage } from '../../utils/fluent.js'
+
+/**
+ * @typedef {object} PartialData
+ * @property {string} csrfToken
+ * @property {number} emailLimit
+ */
+
+/**
+ * @param {PartialData} data
+ * @returns string
+ */
+export const cancelSubscriptionSurvey = () => `
+<div class='dialog'>
+<header>
+  <button class='close'></button>
+  <img src='/images/dialog-unsubscribe-plane.svg'>
```

Comment:
not sure if we want to add `alt=""` or something to this.
We aren't using JSX/TSX, so we cant use the https://www.npmjs.com/package/eslint-plugin-jsx-a11y plugin, but probably safe to not have a translated alt tag for this, per:
> If an image is used simply for decoration and provides no substantive meaning to the page, the `alt` attribute can be set to the empty string. `alt=””` This tells screen readers to skip this image.